### ISSUE TITLE
asHTML renders hyperlinks with undefined target

### ISSUE THREAD

**Issue by seanlail** [prismicio/prismic-helpers#58]
### Versions

- @prismicio/helpers: 2.3.3
- node: v16.13.1

### Reproduction

Using `asHTML`  with the default serializer renders hyperlinks with undefined targets.

### Steps to reproduce

- have a RichTextfield with a hyperlink somewhere. e.g. 
- do not set the target
- in your code use the asHTML helper: `asHTML(myRTF)`

### What is expected?

Hyperlinks should render correctly:
```
<a href="mailto:test@test.com">test@test.com</a>
```

### What is actually happening?

The string is undefined: https://github.com/prismicio/prismic-helpers/blob/master/src/lib/serializerHelpers.ts#L75
```
<a href="mailto:test@test.com" target="undefined" rel="noopener noreferrer">test@test.com</a>
```

The actual code here is different to the suggested example:
https://prismic.io/docs/technologies/html-serializer#see-all-html-serializer-elements

**Comment by angeloashmore** [comment]
Thanks for the report, @seanlail!

I opened a PR fixing this issue here: #59.

Before merging and publishing, I'd like to get a review from @lihbr which will likely happen next week.

If this bug is currently blocking you, however, I'll push it through and publish. Let me know! 🙂

**Comment by angeloashmore** [comment]
@seanlail This bug should be fixed as of `v2.3.4`. For more details on the fix, see #59.

You can update to the latest version with the following command:

```sh
npm install @prismicio/helpers@latest
```

If this doesn't resolve the issue, let me know and I'll take another look. Thanks!
